server/src/main/java/org/opensearch/plugins/SearchPlugin.java

```diff
@@ -138,6 +140,15 @@ default Map<String, Highlighter> getHighlighters() {
         return emptyMap();
     }
 
+    /**
+     * Allows plugins to register custom decider for concurrent search
+     * @return A {@link ConcurrentSearchDecider}
+     */
+    @ExperimentalApi
+    default ConcurrentSearchDecider getConcurrentSearchDeciders() {
+        return null;
+    }
```

Comment:
```suggestion
    /**
     * Allows plugins to register custom decider for concurrent search
     * @return A {@link ConcurrentSearchDecider}
     */
    @ExperimentalApi
    default ConcurrentSearchDecider getConcurrentSearchDecider() {
        return null;
    }
```

Really small suggestion...since it's technically singular now